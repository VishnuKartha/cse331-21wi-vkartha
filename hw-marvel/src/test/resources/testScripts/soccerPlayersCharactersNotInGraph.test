# Tests that the unknown character functionality

# To run this test, the soccer dataset must exist at
#   hw-marvel/src/main/resources/data/soccer.tsv

LoadGraph soccer soccer.tsv

FindPath soccer Kobe_Bryant Cristiano_Ronaldo
FindPath soccer Kobe_Bryant Kobe_Bryant
FindPath soccer Kobe_Bryant Michael_Jordan




