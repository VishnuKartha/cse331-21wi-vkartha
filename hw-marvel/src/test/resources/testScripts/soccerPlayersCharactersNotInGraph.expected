# Tests that the unknown character functionality

# To run this test, the soccer dataset must exist at
# hw-marvel/src/main/resources/data/soccer.tsv

loaded graph soccer

unknown character Kobe Bryant
unknown character Kobe Bryant
unknown character Kobe Bryant
unknown character Kobe Bryant
unknown character Michael Jordan